ee/tabby-webserver/src/server.rs

```diff
@@ -40,6 +42,10 @@ impl ServerContext {
         }
     }
 
+    pub fn auth(&self) -> impl AuthenticationService {
+        self.db_conn.clone()
```

Comment:
is this clone operation cheap to call for every request?

If not - consider returns a reference with lifecycle annotation

Reply:
`db_conn` is just wrapper of `Arc<Connection>`, so `clone` is cheap operation.